app/src/main/res/layout/listitem_weapon.xml

```diff
@@ -3,14 +3,19 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:padding="@dimen/padding_large">
+    android:layout_height="120dp"
+    android:paddingBottom="@dimen/margin_medium"
+    android:paddingEnd="@dimen/margin_medium"
+    android:paddingTop="@dimen/margin_medium">
 
     <TextView
         android:id="@+id/weapon_name"
-        style="@style/TextAppearance.AppCompat.Headline"
+        style="@style/TextHeadlineHigh"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_height="25dp"
+        android:textSize="16sp"
```

Comment:
If you're using the right style then you shouldn't have to set `textSize`. Layout height for textviews should be wrap_content as well.

Reply:
What would be the right text style for this? None of them seem to match the one in your mock